pkg/subctl/cmd/gather.go

```diff
@@ -106,7 +104,6 @@ func gatherData() {
 	submariner := getSubmarinerResource(restConfig)
 	if submariner == nil {
 		fmt.Println(submMissingMessage)
-		return
```

Comment:
This change isn't necessary - this code was removed by https://github.com/submariner-io/submariner-operator/pull/1202 anyway.